lib/beakerhelper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
```

Comment:
Just a little thing, but should this be named `beaker_helper` to adhere to the naming convention used in the other helpers (`abs_helper`, `perf_run_helper`, etc...)?

Reply:
It is a different animal.  It is not designed to contain any additional code for more helpers.  It is merely a wrapper class in order to facilitate creating an instance with the existing beaker DSL Helpers available to it.  If it was made to look like the helper modules, I would be concerned that its intention would be misconstrued.

Reply:
It seems that if the intention is to avoid confusion with the other helper modules that may be better achieved by using a different word than "helper" in the name rather than not using an underscore between the words in the name. The current name doesn't make me think it is different than the other helper modules, it just makes me wonder why it doesn't follow the naming convention used elsewhere in GPLT.

Reply:
Updated.